synfig-studio/ui/canvas_properties.glade

```diff
@@ -0,0 +1,1367 @@
+    <property name="value">1</property>
+    <property name="step_increment">0.10000000000000001</property>
+    <property name="page_increment">0.02</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_gamma_red">
+    <property name="lower">0.10000000000000001</property>
+    <property name="upper">3</property>
+    <property name="value">1</property>
+    <property name="step_increment">0.10000000000000001</property>
+    <property name="page_increment">0.02</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_height">
+    <property name="lower">1</property>
+    <property name="upper">2147483647</property>
+    <property name="value">1</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_phy_h">
+    <property name="upper">10000000</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_phy_w">
+    <property name="upper">10000000</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_span">
+    <property name="upper">10000000</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_width">
+    <property name="lower">1</property>
+    <property name="upper">2147483647</property>
+    <property name="value">1</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_xres">
+    <property name="upper">10000000</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment_yres">
+    <property name="upper">10000000</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkImage" id="toggle_res_img">
+    <property name="visible">True</property>
+    <property name="can_focus">False</property>
+    <property name="stock">synfig-utils_chain_link_off</property>
+    <property name="icon_size">3</property>
+  </object>
+  <object class="GtkImage" id="toggle_wh_img">
+    <property name="visible">True</property>
+    <property name="can_focus">False</property>
+    <property name="stock">synfig-utils_chain_link_off</property>
+    <property name="icon_size">3</property>
+  </object>
+  <object class="GtkBox" id="inner_box">
+    <property name="visible">True</property>
+    <property name="can_focus">False</property>
+    <property name="orientation">vertical</property>
+    <child>
+      <object class="GtkAlignment">
```

Comment:
According to docs: 
```
 gtk_alignment_new has been deprecated since version 3.14 and should not be used in newly-written code.
Use GtkWidget alignment and margin properties
```

Reply:
Strange, I use the verify option in Glade. Didn't report anything about deprecated GtkAlignment object

Reply:
Strange indeed

Reply:
Removed deprecated GtkAlignment from glade
commit 0e6f409c8e6e6052dcdb452e13d2c09dcb39d69d